File: flutter-main/flutter-main/Inheritence/p16.dart
/// class with mixin:-
 

abstract mixin class Demo{
	void fun1(){
		print("In fun1 Demo");
	}
	void fun2();
}
class Asch{
	void ashi(){
		print("In ashi");
	}
}
class child extends Asch with Demo{
	void fun2(){
		print("In fun2 child");
	}
}
void main(){
	child obj = new child();
	obj.fun1();
	obj.fun2();
	obj.ashi();
}

File: java-main/java-main/NestedforloopAssi/p16.java
/*
9
9 8
9 8 7
9 8 7 6
*/

class Demo{

	public static void main(String[]args){

		int N=4;

		for(int i=1;i<=N;i++){
			int num=9;
			for(int j=1;j<=i;j++){
				System.out.print(num-- + " ");
			}
			System.out.println();
		}
	}
}
